pandas/util/testing.py

```diff
@@ -2667,3 +2667,26 @@ def patch(ob, attr, value):
             delattr(ob, attr)
         else:
             setattr(ob, attr, old)
+
+
+@contextmanager
+def set_timezone(tz):
+    import os
```

Comment:
add a doc-string with Parameters here (it must also be a string, so you should str(tz) somewhere.